crates/c/src/lib.rs

```diff
@@ -63,6 +64,10 @@ pub struct Opts {
     /// Rename the interface `K` to `V` in the generated source code.
     #[cfg_attr(feature = "clap", arg(long, name = "K=V", value_parser = parse_rename))]
     pub rename: Vec<(String, String)>,
+
+    /// Disable the autodropping of borrows in exports.
+    #[cfg_attr(feature = "clap", arg(long, default_value_t = false))]
+    pub no_export_autodrop: bool,
```

Comment:
Could this perhaps be phrased in the positive such as `export_autodrop: Option<bool>`?